flask_server.py

```diff
@@ -85,3 +73,14 @@ def cli_check_data_types_all():
 @click.option('-w', '--onlythisweek', is_flag=True)
 def check_cumulative_data(outfile, onlythisweek):
     check_cumulative.cli_check_cumulative_data(outfile, onlythisweek)
+
+
+@app.cli.command("process")
+@click.option('--state', default='', help='State abbreviation to run for, e.g. "CA"')
+@click.option('--write-to-final', is_flag=True)
+@click.option('--other-sheet-url', default='', help='Other sheet URL to write processed file to; '
+    'use if writing to something other than final sheet')
+@click.option('--local-path', default='', help='Local CSV path to write processed file to')
+def process(state, write_to_final, other_sheet_url, local_path):
+    process_module.cli_process_state(state, write_to_final=write_to_final,
+        other_sheet_url=other_sheet_url, local_path=local_path)
```

Comment:
What would you think about reworking a few of these for consistency and documentation? Something broadly like (apparently, if you add a docstring to the function, it shows up in the help):

```suggestion
@app.cli.command("process")
@click.option('--state', default='', help='State abbreviation to run for, e.g. "CA"')
@click.option('--write-to-gsheets', is_flag=True, help='Write to Google Sheets')
@click.option('--out-sheet-url', default='', help='Write the processed data to the specified Google Sheet url')
@click.option('--outfile, default='', help='Write the processed data to CSV file at this local path')
def process(state, write_to_final, other_sheet_url, local_path):
    """Process all data from STATE as defined in the Sheet of Sheets. The processing functions applied to each state are defined in app/api/process.py. 
    The resulting output can be saved to the final sheet defined in the Sheet of Sheets (--write-to-gsheets), to a specified sheet url (--out-sheet-url), to a local CSV file (--outfile), or to STDOUT (default).
    """
    process_module.cli_process_state(state, write_to_final=write_to_final,
        other_sheet_url=other_sheet_url, local_path=local_path)
```

It might make sense to add an `--in-sheet-url` for testing someday, but no need unless we want it.

Reply:
chatted about this offline, did something similar to what's suggested here!